<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interative segmentation with Superpixels</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>777</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolTip">
          <string>back</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolTip">
          <string>forward</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/forward.png</normaloff>icons/forward.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="eraseSuperpixel">
       <property name="toolTip">
        <string>erase</string>
       </property>
       <property name="text">
        <string>Eraser</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/clean.png</normaloff>icons/clean.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paintSuperpixel">
       <property name="toolTip">
        <string>paint superpixel</string>
       </property>
       <property name="text">
        <string>Pencil</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/draw.png</normaloff>icons/draw.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leastPath">
       <property name="toolTip">
        <string>line path</string>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/path.png</normaloff>icons/path.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dynamicSegmentation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Dynamic</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/bucket.png</normaloff>icons/bucket.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="regionGrowing">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>bucket tool: region growing</string>
       </property>
       <property name="text">
        <string>Growing</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/bucket.png</normaloff>icons/bucket.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forestSegmentation">
       <property name="toolTip">
        <string>bucket tool: forest segmentation</string>
       </property>
       <property name="text">
        <string>Forest segm</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/bucket.png</normaloff>icons/bucket.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderOpacity">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
       <property name="sliderPosition">
        <number>80</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="opacityValueText">
       <property name="text">
        <string>80</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tolerance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderTolerance">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="toleranceValueText">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>620</y>
      <width>81</width>
      <height>72</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/zoom_out.png</normaloff>icons/zoom_out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>540</y>
      <width>81</width>
      <height>72</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/zoom_in.png</normaloff>icons/zoom_in.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Superpixel_file"/>
    <addaction name="actionSave_mask"/>
   </widget>
   <widget class="QMenu" name="menuSuperpixels">
    <property name="title">
     <string>Superpixels</string>
    </property>
    <addaction name="actionISF"/>
    <addaction name="actionSLIC"/>
   </widget>
   <widget class="QMenu" name="menuSeeds">
    <property name="title">
     <string>Seeds</string>
    </property>
    <addaction name="actionGenerate_seeds"/>
    <addaction name="actionLoad_seeds_from_CSV"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicence"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSuperpixels"/>
   <addaction name="menuSeeds"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Superpixel_file">
   <property name="text">
    <string>Open Superpixel file</string>
   </property>
  </action>
  <action name="actionGenerate_Superpixels">
   <property name="text">
    <string>Generate Superpixels</string>
   </property>
  </action>
  <action name="actionLoad_seeds_from_CSV">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load seeds from CSV</string>
   </property>
  </action>
  <action name="actionSave_mask">
   <property name="text">
    <string>Save mask</string>
   </property>
  </action>
  <action name="actionISF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ISF</string>
   </property>
  </action>
  <action name="actionSLIC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SLIC</string>
   </property>
  </action>
  <action name="actionGenerate_seeds">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Generate seeds</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionLicence">
   <property name="text">
    <string>Licence</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
